README.md

```
@@ -1,5 +1,293 @@
-# Chat-bot-generador-de-imagenes
+# Chat-bot-generador-de-promts
+#!/usr/bin/env python3
```

markdownlint-cli2 (0.17.2)

No space after hash on atx style heading

(MD018, no-missing-space-atx)